tf-psa-crypto/.gitignore

```diff
@@ -0,0 +1 @@
+Makefile
```

Comment:
`t<TAB>` doesn't work anymore :( I don't know if there's a sensible solution.

Reply:
no you have to move to `te<TAB>`.